### ISSUE TITLE
[A11y] 1.4.11 Non-text Contrast - Lack of Contrast UI Components

### ISSUE THREAD

**Issue by dalonsoa** [research-software-directory/RSD-as-a-service#1768]
See https://github.com/ImperialCollegeLondon/RSD-as-a-service/issues/38 for details.

This is on us, but I add it here for completeness as it might affect other pages from other people.

**Comment by dalonsoa** [comment]
Mostly sorted, but still some areas of improvement. Will work on them over the coming weeks. It has minor impact on accessibility, it would seem, so it should be OK.
